Assets/Scripts/Prototype/ProjectilePrototype/SpawnProjectile.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Controls;
+
+public class SpawnProjectile : MonoBehaviour
+{
+    public static bool didProjectileHit = false;
+
+    public GameObject firePoint;
+    public List<GameObject> vfx = new List<GameObject>();
+    public Camera currentCamera;
+    public AudioSource shootSound;
+    public AudioSource impactSound;
+
+    private GameObject effectToSpawn;
+    private float timeToFire = 0;
+
+    void Start()
+    {
+        AetherInput.GetPlayerActions().Fire.performed += SpawnVFX;
+        effectToSpawn = vfx[0];
```

Comment:
This may throw an indexOutOfBound exception

Reply:
true i hardcoded the size but i realized i dont need to make an array